Sources/Protocols/MessagesLayoutDelegate.swift

```diff
@@ -22,6 +22,31 @@
  SOFTWARE.
  */
 
+public struct AvatarPosition {
+    
+    public enum Horizontal {
+        case cellLeading
+        case cellTrailing
```

Comment:
Here is where I'm talking about the `.natural` case

Reply:
I think `.natural` would be good and have it be default